### PR TITLE
[tests]: Set HWSKU to NPU for dash vstests

### PR DESCRIPTION
**Pull request by prabhataravind**
**What I did**
Define a generic NPU SKU for docker-sonic-vs
**Why I did it**
Bluefield3 hwsku(Nvidia-MBF2H536C) is not planned to be made available upstream in the near future. Therefore , inorder to pass DASH vstests on azure pipeline runs, this change is being made.
**How I verified it**
By making sure that DASH vstests are able to run and pass on azure pipeline.
**Details if related**

### CODE REVIEW

File: tests/test_dash_acl.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 import pytest
 
-DVS_ENV = ["HWSKU=Nvidia-MBF2H536C"]
+DVS_ENV = ["HWSKU=NPU"]
```
Comment (prsunny) [inline tests/test_dash_acl.py:8]: NPU or DPU?

Comment (prabhataravind) [reply tests/test_dash_acl.py:8]: Either one is ok, but Guohan suggested NPU.

**Comment by prsunny** [comment]
Can you please link the sonic-buildimage PR here?

**Review by prsunny — APPROVED** [review]

**Comment by prabhataravind** [comment]
https://github.com/sonic-net/sonic-buildimage/pull/15604

**Pull request merged**